src/app/features/search/components/search-form/search-form.component.ts

```diff
@@ -85,12 +85,12 @@ export class SearchFormComponent implements OnInit {
   public ngOnInit() {
     const { from, to } = this.searchForm.controls;
```

Comment:
we use a getter for `controls` so we can use it here ?
```suggestion
    const { from, to } = this.controls;
```